src/test/java/org/jitsi/videobridge/cc/SeenFrameTest.java

```diff
@@ -0,0 +1,93 @@
+package org.jitsi.videobridge.cc;
+
+import org.easymock.*;
+import org.jitsi.impl.neomedia.rtp.*;
+import org.jitsi.service.neomedia.*;
+import org.junit.*;
+import org.junit.experimental.runners.*;
+import org.junit.runner.*;
+import org.junit.runners.*;
+
+import java.util.*;
+
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.isA;
+import static org.junit.Assert.*;
+
+/**
+ * Created by bbaldino on 8/9/17.
+ */
+@RunWith(Enclosed.class)
+public class SeenFrameTest
+    extends EasyMockSupport
+{
+    @RunWith(Parameterized.class)
+    public static class SeenFrameAcceptTest
+        extends EasyMockSupport
+    {
+        @Parameterized.Parameters
+        public static Collection<Object[]> data()
+        {
+            return Arrays.asList(new Object[][] {
+                //  frameMinSeen,   frameMaxSeen,   pktSeqNum,  isMostRecentFrame,  isAdaptive, expectedResult
+                // Current frame, non-adaptive stream, sequence number in range
+                {   5,              10,             7,          true,               false,      true },
+                // Current frame, non-adaptive stream, sequence number not in range
+                {   5,              10,             12,         true,               false,      false },
+                // Current frame, adaptive stream, sequence number in range
+                {   5,              10,             7,          true,               true,       true },
+                // Current frame, adaptive stream, sequence number not in range
+                {   5,              10,             12,         true,               true,       false },
+                // Old frame, non-adaptive stream, sequence number in range
+                {   5,              10,             7,          false,              false,      true },
+                // Old frame, non-adaptive stream, sequence number not in range
+                {   5,              10,             12,         false,              false,      false },
+                // Old frame, adaptive stream, sequence number in range
+                {   5,              10,             7,          false,              true,       false },
+                // Old frame, adaptive stream, sequence number not in range
+                {   5,              10,             12,         false,              true,       false },
```

Comment:
you mean specifically the values `5`, `10`, `7` and `12`?  i think that'd be a bit weird because i don't know what i'd name them (the only thing interesting about them is their relationship to one another, other than that they're totally arbitrary).  Even if they were named, i'd still keep the comment above each describing the scenario at a higher level.  i did think about:
1) turning the first 3 values into a boolean `packetWithinFrame` which would be true or false.  this then corresponds to the way it's described in the comment, but then the test would need to translate that into a set of 3 sequence numbers to meet that criteria didn't feel all that much better than what it is now
2) putting the 3 seq num values in a mini-class and having one called `packetWithinFrame` and another `packetOutsideFrame` but this is pretty much the same as the first idea, just executed differently.

neither seemed great enough to replace what was there.